.changeset/blue-otters-joke.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏this needs a changeset because non-test code did change, just in case

Reply:
It _is_ a no-op change in this case, since we know it is. But I don't mind having a hard-line stance as well.